rln/Cargo.toml

```
@@ -11,6 +11,8 @@ doctest = false
 
 
 [dependencies]
+pmtree = { git = "https://github.com/Rate-Limiting-Nullifier/pmtree" }
```

What do you think of locking the pmtree version by using a commit, or maybe publishing `pmtree` so if a breaking change is introduced, we can still build zerokit? 
```suggestion
pmtree = { git = "https://github.com/Rate-Limiting-Nullifier/pmtree"  rev = "14d58fd" }
```

---

I think @s1fr0 just forgot to do it, It's always a good practice to lock on commits.